### PR TITLE
fix: default image size

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes mf-4139

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/web3-shared/evm/src/helpers/resolver.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export function resolveImageURL(image?: string, name?: string, collectionName?:
 
 export function isLens(name?: string) {
     if (!name) return false
-    return name.toLowerCase().endsWith('.lens') || name.toLowerCase() === 'lensprotocol'
+    return name.toLowerCase().endsWith('.lens') || name.toLowerCase().includes('lensprotocol')
```
Comment (guanbinrui) [inline packages/web3-shared/evm/src/helpers/resolver.ts]: CC @UncleBill

Comment (UncleBill) [reply packages/web3-shared/evm/src/helpers/resolver.ts]: @lelenei Any case to explain this?

Comment (lelenei) [reply packages/web3-shared/evm/src/helpers/resolver.ts]: @UncleBill  name is @lesprotocol

**Review by guanbinrui — APPROVED** [review]

File: packages/web3-shared/evm/src/helpers/resolver.ts (hunk 1 above)
Comment (guanbinrui) [reply packages/web3-shared/evm/src/helpers/resolver.ts]: Does `=== '@lensprotocol'` a better choice?

**Review by UncleBill — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**